src/user/serializers.py

```diff
@@ -457,6 +459,12 @@ def get_author_profile(self, user):
             sentry.log_error(e)
             return {}
 
+    def get_rsc_earned(self, user):
+        return user.rsc_earned
+
+    def get_benefits_expire_on(self, user):
+        return user.benefits_expire_on
```

Comment:
```suggestion
        benefits_expire_on = getattr(user, "benefits_expire_on", None)
        return benefits_expire_on
```
This ensures that it won't error out

Reply:
ah good good